torch/csrc/jit/script/init.cpp

```diff
@@ -177,6 +187,28 @@ struct VISIBILITY_HIDDEN PythonModuleValue : public PythonValue {
   }
 };
 
+// For ignored Python functions, set the PythonOp in the graph to
+// be ignored on export
+struct VISIBILITY_HIDDEN IgnoredPythonValue : public PythonValue {
+  IgnoredPythonValue(py::object self) : PythonValue(std::move(self)) {}
+
+  std::string kind() const override {
+    return "ignored python value";
+  }
+
+protected:
+  Node* getPythonOp(
+      const std::shared_ptr<Graph>& graph,
+      THPObjectPtr func_ptr,
+      std::string& cconv) const override {
+    Node* node = graph->insertNode(
+        graph->createPythonOp(std::move(func_ptr), cconv, {}));
```

Comment:
nit: you can call `PythonValue::getPythonOp` to avoid spelling this out again (but leaving as is is fine with me too).